tests/js/spec/views/dataExport/dataDownload.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import DataDownload, {DownloadStatus} from 'app/views/dataExport/dataDownload';
+
+describe('DataDownload', function() {
+  beforeEach(MockApiClient.clearMockResponses);
+  const dateExpired = new Date();
+  const mockRouteParams = {
+    orgId: 'acme',
+    dataExportId: 721,
+  };
+  const getDataExportDetails = body =>
+    MockApiClient.addMockResponse({
+      url: `/organizations/${mockRouteParams.orgId}/data-export/${mockRouteParams.dataExportId}/`,
+      body,
+    });
+
+  it('should send a request to the data export endpoint', function() {
+    const getValid = getDataExportDetails(DownloadStatus.Valid);
+    mountWithTheme(<DataDownload params={mockRouteParams} />);
+    expect(getValid).toHaveBeenCalled();
+  });
+
+  it("should render the 'Early' view when appropriate", function() {
+    const status = DownloadStatus.Early;
+    getDataExportDetails({status});
+    const wrapper = mountWithTheme(<DataDownload params={mockRouteParams} />);
+    expect(wrapper.state('download')).toEqual({status});
+    expect(wrapper.state('download').dateExpired).toBeUndefined();
+    const contentWrapper = wrapper.find('div[data-test="datadownload-wrapper"]');
```

Comment:
you can use component names with enzyme e.g. wrapper.find('ContentContainer')

Reply:
I saw that Airbnb doesn't recommend that, but im not about to export a styled component -- done in f387c4da13

Reply:
Oh? Can you link me?